### PR TITLE
Create two and three variable modelling subdialogues

### PR DESCRIPTION
**Pull request by stevekogo**

### CODE REVIEW

**Comment by dannyparsons** [comment]
I think there should only be one model options dialog. It might look different when it's called from different dialogs but we can control this with the user controls that will go on it.

**Comment by stevekogo** [comment]
@dannyparsons I deleted the two sub-dialogues and kept only one.

**Comment by dannyparsons** [comment]
Great can you then rename this dialog to ModelOptions.

**Comment by stevekogo** [comment]
@dannyparsons I have re-created the dialogue

File: instat/instat.vbproj [hunk 1]
```diff
@@ -3465,6 +3471,15 @@
...
     </EmbeddedResource>
+    <EmbeddedResource Include="sdgTwoVarModelOptions.fr-FR.resx">
+      <DependentUpon>sdgTwoVarModelOptions.vb</DependentUpon>
+    </EmbeddedResource>
+    <EmbeddedResource Include="sdgTwoVarModelOptions.resx">
+      <DependentUpon>sdgTwoVarModelOptions.vb</DependentUpon>
+    </EmbeddedResource>
+    <EmbeddedResource Include="sdgTwoVarModelOptions.sw-KE.resx">
+      <DependentUpon>sdgTwoVarModelOptions.vb</DependentUpon>
+    </EmbeddedResource>
```
Comment (dannyparsons) [inline instat/instat.vbproj]: Something's not quite right here because in vbproj it's still called sdgTwoVarModelOptions. Maybe need to update this file?

**Comment by stevekogo** [comment]
@dannyparsons I am closing this pull request. I have made a new pull request with model options subdialogue.

**Pull request closed without merging**